src/main/kotlin/com/nikitahohulia/listeningplatform/config/Config.kt

```diff
@@ -0,0 +1,13 @@
+package com.nikitahohulia.listeningplatform.config
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class Config {
+
+    @Bean
+    fun logger(): Logger = LoggerFactory.getLogger(Logger::class.java)
+}
```

Comment:
this is bad practice to create a logger. logger should be bound to some class, so create it in companion object of your BPP